onto-viewer-core/src/main/java/org/edmcouncil/spec/ontoviewer/core/ontology/loader/AutoOntologyLoader.java

```diff
@@ -138,7 +138,7 @@ private void loadMappersToOntologyManager(OWLOntologyManager manager) {
             ontologyPath = ontologyMappingParentPath.resolve(ontologyPath).normalize();
           }
           if (Files.exists(ontologyPath)) {
-            ontologyMapping.put(mapping.getName(), ontologyPath.toString());
+            ontologyMapping.put(ontologyPath.getFileName().toString(), ontologyPath.toString());
```

Comment:
This looks wrong - ontologyMapping is to store the mappings from ontologies iris to ontologies locations in a local file system.